tests/setup.ts

```diff
@@ -52,8 +40,14 @@ export async function setup(): Promise<{
   attestation: Kilt.Attestation
   attestedClaim: Kilt.AttestedClaim
 }> {
+  await Kilt.init({ address: 'wss://full-nodes.kilt.io:9944' })
```

Comment:
I don't think we want to run the workflow tests against the testnet? we need an identity which definitely has tokens, which is why we have tested against a development node so far afaik